djvm/src/main/kotlin/net/corda/djvm/analysis/AnalysisConfiguration.kt

```diff
@@ -126,12 +116,15 @@ class AnalysisConfiguration private constructor(
         const val SANDBOX_PREFIX: String = "sandbox/"
 
         /**
-         * These class must belong to the application class loader.
+         * An empty placeholder used by "child" instances of [SourceClassLoader].
+         */
+        private val EMPTY: BootstrapClassLoader = BootstrapClassLoader()
+
+        /**
+         * These classes must belong to the application class loader.
          * They should already exist within the sandbox namespace.
          */
-        private val MANDATORY_PINNED_CLASSES: Set<String> = setOf(
-            ruleViolationError
-        )
+        private val MANDATORY_PINNED_CLASSES: Set<String> = emptySet()
```

Comment:
Should we just get rid of `MANDATORY_PINNED_CLASSES` ?

Reply:
:+1: